kowalski/alert_broker_ztf.py

```diff
@@ -892,7 +892,33 @@ def __init__(self, **kwargs):
                 raise ValueError("Failed to get ZTF instrument_id from SkyPortal")
         except Exception as e:
             log(e)
-            # config['misc']['broker'] = False
+
+        # get ZTF alert stream ids to program ids mapping
+        self.ztf_program_id_to_stream_id = {0: 3, 1: 1, 2: 2, 3: 3}
```

Comment:
```suggestion
        self.ztf_program_id_to_stream_id = dict()
```
? And then error if all three expected streams aren't found in the response below? otherwise we're still defaulting to these possibly-wrong stream IDs

Reply:
Right... I thought about defaulting to posting everything to the Sitewide group, but I guess if this operation failed, things are pretty bad and it's a good idea to throw an error instead. 
On the other hand, if dask spawns a new worker process while SP is unresponsive (which as we know could sometimes happen), the current code wouldn't err at init, but rather only when posting photometry to SP. I removed try-except - I think it's safer to kill the worker and re-try later.